### PR TITLE
Added gymclassinstance.py and instructor.py

### PR DESCRIPTION
**Pull request by williambaibai**
screen shot 2018-03-01 at 10 45 33 pm

`created gymclassinstance.py and instructor.py` and `added columns 'instructor_id'(fk), 'location', 'name' to gymclass.py`

### CODE REVIEW

File: src/app/classes/models/_all.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from app.classes.models.user import *
 from app.classes.models.gym import *
 from app.classes.models.gymclass import *
+from app.classes.models.instructor import *
+from app.classes.models.gymclassinstance import *
```
Comment (young-k) [inline src/app/classes/models/_all.py]: really minor, but can you alphabetize these imports: i.e. `gym` at the top, `user` at the bottom

File: src/app/classes/models/_all.py [hunk 2]
```diff
@@ -14,3 +16,11 @@ class Meta(ModelSchema.Meta):
 class GymClassSchema(ModelSchema):
  class Meta(ModelSchema.Meta):
    model = GymClass
+
+class InstructorSchema(ModelSchema):
```
Comment (young-k) [inline src/app/classes/models/_all.py]: same with these `schema` exports

File: src/app/classes/models/gymclass.py [hunk 3]
```diff
@@ -5,19 +5,33 @@ class GymClass(Base):
...
 
   gym = db.relationship('Gym', backref='gyms')
+
   description = db.Column(db.String(2000))
 
+  instructor_id = db.Column(db.Integer,
+                            db.ForeignKey('instructors.id', ondelete='CASCADE'))
+  instructor = db.relationship('Instructor', backref='instructors')
+
+  location = db.Column(db.String(500))
```
Comment (young-k) [inline src/app/classes/models/gymclass.py]: do we need this field if `gym` stores the location? don't feel like we do

File: src/app/classes/models/gymclass.py [hunk 4]
```diff
@@ -5,19 +5,33 @@ class GymClass(Base):
...
   gym_id = db.Column(db.Integer,
                      db.ForeignKey('gyms.id', ondelete='CASCADE'))
 
   gym = db.relationship('Gym', backref='gyms')
+
   description = db.Column(db.String(2000))
 
+  instructor_id = db.Column(db.Integer,
+                            db.ForeignKey('instructors.id', ondelete='CASCADE'))
+  instructor = db.relationship('Instructor', backref='instructors')
```
Comment (young-k) [inline src/app/classes/models/gymclass.py]: shouldn't this `backref` be `gym_classes`

File: src/app/classes/models/instructor.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+from . import *
+
+class Instructor(Base):
+  __tablename__ = 'instructors'
+
+  id = db.Column(db.Integer, primary_key=True)
+  first_name = db.Column(db.String(255), nullable=False)
+  last_name = db.Column(db.String(255))
+  email = db.Column(db.String(255), unique=True)
+
```
Comment (young-k) [inline src/app/classes/models/instructor.py]: even more minor, can you delete this newline

File: src/app/classes/models/gymclass.py (hunk 3 above)
Comment (williambaibai) [reply src/app/classes/models/gymclass.py]: I was thinking if we need a field to store like the room number of the class? Let me know!

**Pull request merged**